plugin-build/build.gradle.kts

```diff
@@ -70,3 +70,34 @@ if (gradle.gradleVersion >= "6.6.0") {
     val publish = extensions.getByType(MavenPublishPluginExtension::class.java)
     publish.releaseSigningEnabled = BuildUtils.shouldSignArtifacts()
 }
+
+subprojects {
+    apply<DistributionPlugin>()
+
+    val sep = File.separator
+
+    configure<DistributionContainer> {
+        this.getByName("main").contents {
+            from("build${sep}libs")
+            from("build${sep}publications${sep}maven")
+            from("build${sep}publications${sep}sentryPluginPluginMarkerMaven")
```

Comment:
```suggestion
            from("build${sep}publications${sep}sentryPluginPluginMarkerMaven")
            duplicatesStrategy = DuplicatesStrategy.INCLUDE
```

Reply:
that would fix https://github.com/getsentry/sentry-android-gradle-plugin/pull/145/files#r669520898 I guess